marklogic-data-hub/src/test/java/com/marklogic/hub/flow/FlowRunnerTest.java

```diff
@@ -73,18 +73,14 @@ public void setupEach() {
     public void testRunFlow(){
         RunFlowResponse resp = fr.runFlow("testFlow");
         fr.awaitCompletion();
+        System.out.println("Logging response to help with debugging this failure on Jenkins: " + resp);
         Assertions.assertTrue(getDocCount(HubConfig.DEFAULT_STAGING_NAME, "xml-coll") == 1);
         Assertions.assertTrue(getDocCount(HubConfig.DEFAULT_STAGING_NAME, "csv-coll") == 25);
         Assertions.assertTrue(getDocCount(HubConfig.DEFAULT_STAGING_NAME, "csv-tab-coll") == 25);
         Assertions.assertTrue(getDocCount(HubConfig.DEFAULT_STAGING_NAME, "json-coll") == 1);
         Assertions.assertTrue(getDocCount(HubConfig.DEFAULT_FINAL_NAME, "json-map") == 1);
         Assertions.assertTrue(getDocCount(HubConfig.DEFAULT_FINAL_NAME, "xml-map") == 1);
         Assertions.assertTrue(JobStatus.FINISHED.toString().equalsIgnoreCase(resp.getJobStatus()));
-        Assertions.assertNotNull(resp.getUser());
```

Comment:
Are these tests removed temporarily until we find the underlying issue ?

Reply:
I moved them to a unit test that verifies that data is copied from one RunFlowResponse to another.